llama_stack/cli/stack/run.py

```diff
@@ -75,6 +80,31 @@ def _add_arguments(self):
             help="Image Type used during the build. This can be either conda or container or venv.",
             choices=[e.value for e in ImageType],
         )
+        self.parser.add_argument(
+            "--quota-store-type",
+            type=str,
+            choices=[KVStoreType.sqlite.value, KVStoreType.redis.value],
+            default=KVStoreType.sqlite.value,
+            help="KV store type to back per-client quotas",
+        )
+        self.parser.add_argument(
+            "--quota-store-db-path",
+            type=str,
+            default=None,
+            help="If using sqlite KV store, filesystem path to the database file",
+        )
+        self.parser.add_argument(
+            "--quota-requests-per-day",
+            type=int,
+            default=None,
+            help="Max requests per client per day",
+        )
+        self.parser.add_argument(
+            "--quota-window-seconds",
+            type=int,
+            default=None,
+            help="Time window for the daily quota, in seconds",
+        )
```

Comment:
I don't think we want to add any more flag, we just don't have a good policy that right now. So please keep all that as a config fields. Thanks!